src/main/resources/application-dev.yml

```diff
@@ -29,3 +32,8 @@ hmpps:
   prisonersearch:
     api:
       url: "https://prisoner-offender-search-dev.prison.service.justice.gov.uk"
+
+cache:
+  evict:
+    bank-holidays:
+      cron: "0 45 23 * * ?"
```

Comment:
if you declare this in application-yml you don't need to declare it here

Reply:
Removed and declared in one place `application.yml`